packages/stacks-svelte/src/components/EmptyState/EmptyState.stories.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts" module>
+    import { defineMeta } from "@storybook/addon-svelte-csf";
+    import EmptyState from "./EmptyState.svelte";
+    import Link from "../Link/Link.svelte";
+
+    const { Story } = defineMeta({
+        title: "Components/EmptyState",
+        component: EmptyState,
+        tags: ["labs"],
+        argTypes: {
+            title: {
+                control: "text",
+            },
+            class: {
+                control: "text",
+            },
+        },
```

Comment:
These types get lifted from the component automatically, so we can save a few characters here

```suggestion
```